### PR TITLE
Design home and about page

### PR DESCRIPTION
**Pull request by aimalamiri**
# Module 1 Capstone Project
1. Designed the home page for the desktop and mobile screens
2. Designed the about page for the desktop and mobile screens
3. Used Bootstrap 5 Framework
4. Used Bootstrap icons
5. Used Javascript even listener for toggling the mobile menu icon
6. Team members (or speakers in the original design) are dynamically generated in Javascript.
7. Wrote a descriptive README file with a screenshot of the project for mobile and desktop screens.
8. Deployed the project to Github pages

Here is the [documentation loom video link](https://www.loom.com/share/e59055aff0b1406a97f851ee97f93b7e), and here is the link to the [live demo](https://aimalamiri.github.io/CoWorking-Space).

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+            aria-controls="navbarNav"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <a class="navbar-brand d-none d-sm-block" href="./">
+            <img src="./images/logo.png" alt="" height="51" />
+          </a>
+          <div class="collapse navbar-collapse" id="navbarNav">
```
Comment (MeqdamAlqudah) [inline about.html]: - [x] Please make sure to have an option in the navbar that sends the user to the home page 
image I know this is not in the design but you have a home page 
the user should be able to switch between the home and the about page

File: about.html [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        </div>
+        <div class="col-12 col-md-9 px-5 text-gray">
+          <h6 class="fw-bold">2022 CoWorking Space Afghanistan, Some Rights Reserved.</h6>
+          <p>
+            All content on the website, unless otherwise specified, is available under a Creative Commons Attribution
+            2.0 Afghanistan license.
+          </p>
+        </div>
+      </div>
+    </footer>
```
Comment (MeqdamAlqudah) [inline about.html]: - [x] Please make sure that the footer matches Behance design in the mobile version the logo should be aligned to the right and the text to the left
 
| Your design | Behance |
| ------------- | --------- |
| image | image |

File: about.html [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+            <h1 class="text-orange font-cg-regular hero-title text-center">COWORKING SPACE YOUR DREAMS OFFICE</h1>
+            <div class="small-border mx-auto mt-3 mb-5"></div>
+            <div class="hero-text bg-white py-4 text-gray font-cg-light fs-7 text-center border-light">
+              Lorem ipsum dolor sit amet consectetur, adipisicing elit. Accusantium, suscipit molestiae? Velit corrupti
+              numquam libero impedit. Debitis, in? Ipsam soluta error adipisci corrupti saepe est vero id ad
+              necessitatibus voluptates.
+              <p>
+                Lorem ipsum dolor, sit amet consectetur adipisicing elit. Optio hic veniam adipisci, tenetur eaque vitae
+                sed quas nesciunt aspernatur dolore nobis repellat provident, in, ut repellendus similique harum vel
+                non.
```
Comment (MeqdamAlqudah) [inline about.html:104]: - [x] Please make sure that the text matches the one in the Behance design 

| Your design | Behance |
| ------------- | --------- |
| image | image |
| Please make sure that you are using the same font color ,the same text,the same font family (Cocogoose, Lato)  |

File: index.html [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+            <div class="col-md col-12 card facility-card text-center py-4 px-3 rounded-0">
+              <div class="row align-items-center">
+                <span class="col-2 col-md-12 bi bi-people text-white h1"></span>
+                <h5 class="col-4 col-md-12 text-orange my-3">Meeting Room</h5>
+                <p class="col-6 col-md-12 text-white fs-7 my-auto">
+                  Fully furnished and up to 10 people accommodable space for your training sessions, events, and
+                  meetings.
+                </p>
+              </div>
+            </div>
```
Comment (MeqdamAlqudah) [inline index.html:147]: - [x] Please make sure to Implement the hovering technique is in Behance design 

| Your design | Behance |
| ------------- | --------- |
| image | image |
if you notice the design you can see how the border turns to be white when the user do an over mouse event

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @aimalamiri , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: about.html (hunk 1 above)
Comment (aimalamiri) [reply about.html]: In the task, Microverse asked me to give a link to the home page with the logo, so when the user clicks on the logo it'll redirect the user to the homepage.

Comment (MeqdamAlqudah) [reply about.html]: @aimalamiri sure aimalamiri most of the websites send you to the home page when you press the logo(i.e that is why Microverse told you to put a link to the home page when the user presses the logo ), but it is also required to put a home page link so the user can navigate easily between the home page and the about page i.e it should be simple to the user remember KISS principle.

**Review by codepantha — APPROVED** [review]
Hi @aimalamiri 👋

## STATUS: APPROVED ✔️  

You have done well implementing all the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### To highlight 💡 
- Beautiful and responsive UI ✅ 
- Descriptive PR ✅ 
- Meaningfull commit messages ✅ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**